plugins/woocommerce/composer.json

```diff
@@ -21,13 +21,14 @@
 		"maxmind-db/reader": "^1.11",
 		"pelago/emogrifier": "^6.0",
 		"woocommerce/action-scheduler": "3.4.2",
-		"woocommerce/woocommerce-blocks": "9.1.1"
+		"woocommerce/woocommerce-blocks": "9.1.3"
 	},
 	"require-dev": {
 		"bamarni/composer-bin-plugin": "^1.4",
 		"yoast/phpunit-polyfills": "^1.0",
 		"phpunit/phpunit": "7.5.20",
-		"automattic/jetpack-changelogger": "3.1.3"
+		"automattic/jetpack-changelogger": "3.1.3",
+		"sebastian/comparator": "3.0.3"
```

Comment:
Version 3.0.5 of `sebastian/comparator` was causing the testing oddities, so I've pinned to 3.0.3.